src/CommunityToolkit.Maui.MediaElement/Extensions/PageExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace CommunityToolkit.Maui.Extensions;
+
+// Since MediaElement can't access .NET MAUI internals we have to copy this code here
+// https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/Platform/PageExtensions.cs
+static class PageExtensions
+{
+	internal static bool isFullScreen { get; set; } = false;
```

Comment:
I don't think this is actually used anywhere so we could remove it? Or if we need to keep it we might want to Pascal case it

Reply:
Yes I forgot to remove that

Reply:
This has been removed. Marked as resolved.